packages/core/src/actors/transition.ts

```diff
@@ -25,7 +25,7 @@ export type TransitionActorLogic<
   TEmitted
 >;
 
-export type TransitionActorRef<
+type TransitionActorRef<
```

Comment:
Same here; we may want this exported in `index.ts`?

Reply:
Resolved in #4976